pyproject.toml

```diff
@@ -1,40 +1,49 @@
+[build-system]
+requires      = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "darglint2"
 version = "1.8.2"
-description = "A utility for ensuring Google-style docstrings stay up to date with the source code."
-authors = ["terrencepreilly <terrencepreilly@gmail.com>"]
+description = "A utility for ensuring docstrings stay up to date with the source code."
+authors = ["terrencepreilly"]
+maintainers = [
+    "Antti Kaihola <13725+akaihola@users.noreply.github.com>",
+    "yfprojects <real-yfprojects@users.noreply.github.com>",
+]
 license = "MIT"
+repository = "http://github.com/akaihola/darglint2"
+documentation = "https://raw.githubusercontent.com/akaihola/darglint2/master/docs/poster.pdf"
 keywords = ["documentation", "linter", "development"]
 classifiers = [
     'Intended Audience :: Developers',
     'Topic :: Software Development :: Documentation',
     'Topic :: Software Development :: Quality Assurance',
     'License :: OSI Approved :: MIT License',
-    'Programming Language :: Python :: 3.9',
-    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3 :: Only',
```

Comment:
Is l.23 needed?

Reply:
Sorry, what's `l.23`?

Reply:
Ah, line 23. I'd need to check the documentation for classifiers, I don't know.

Reply:
I checked ~15 of the first hits for the [PyPI search on `Python :: 3 :: Only`, and about half of those seem to also include `Python :: 3`. Based on that, I'd say this is purely a matter of taste.

Reply:
will do

Reply:
I couldn't find any specific documentation of the classifiers and how parenting works. Pypis filtering by classifier doesn't work, so I couldn't check through that.

Reply:
I'm happy with this as it is now.